packages/rest/src/lib/errors/HTTPError.ts

```diff
@@ -12,18 +11,19 @@ export class HTTPError extends Error {
 
 	/**
 	 * @param status - The status code of the response
+	 * @param statusText - The status text of the response
 	 * @param method - The method of the request that erred
 	 * @param url - The url of the request that erred
 	 * @param bodyData - The unparsed data for the request that errored
 	 */
 	public constructor(
 		public status: number,
+		public statusText: string,
```

Comment:
```suggestion
		statusText: string,
```

This is the message of the base errror, no need to double up.